packages/core/src/electron-browser/menu/electron-menu-contribution.ts

```diff
@@ -68,9 +69,21 @@ export namespace ElectronMenus {
 @injectable()
 export class ElectronMenuContribution implements FrontendApplicationContribution, CommandContribution, MenuContribution, KeybindingContribution {
 
+    // Specified in window preferences schema
+    protected ZOOM_MIN = -8;
+    protected ZOOM_MAX = 9;
+    // Amount to increment or decrement the window zoom level.
+    protected ZOOM_VARIATION = 0.5;
```

Comment:
Should we define these constants in the `electron-window-preferences`?
They can are used when defining the schema, and here if necessary.